mobile/src/defaultViewComponents/defaultViewTable.js

```diff
@@ -24,31 +25,57 @@ export default class DefaultViewTable extends Component {
 
   render() {
     const { currentFilter, changeTableFilter, reportItem, resolveItem } = this.props
-    const data = this.verifyData()
+    const userData = this.verifyData(true)
+    const data = this.verifyData(false)
     return (
       <View>
         <DefaultViewTableHeader currentFilter={currentFilter} changeTableFilter={changeTableFilter} items={Object.values(this.props.items)}/>
-        <FlatList
+        {userData.length ? <FlatList
+          data={userData}
+          renderItem={({item}) => {
+            const reports = this.props.reports
+            const isReported = ((reports && reports.find(report => report === item.id)) ? true : false)
+            return (
+              <DefaultViewTableCell item={item} reportItem={reportItem} isReported={isReported} resolveItem={resolveItem}/>
+            )
+          }} 
+        /> : null }
+        {Object.values(this.props.items).length || data.length ? <FlatList
```

Comment:
Why does the inclusion of this FlatList depend on both `this.props.items` and `data` but only display the `data`?

Reply:
Actually after more thought I think I only need to check for this.props.items